.github/workflows/run-lint.yml

```
@@ -0,0 +1,22 @@
+name: Tests
```

We might want to rename this to avoid confusion with the other `Tests` workflow

```suggestion
name: Lint
```

---

Thought i did! Good catch